src/realizaJogadas.cpp

```diff
@@ -273,5 +292,41 @@ bool venceu(char tabuleiro[][SIZE]) {
 	return flag;
 }
 
+bool jogadaSeraAutomatica() {
+
+	char escolha;
+
+	do {
+
+		printf("Deseja que a proxima jogada seja automatica?\nS - SIM    N - NAO\n");
+		getchar();
+		scanf("%c", &escolha);
+
+		escolha = toupper(escolha);
+
+	} while(escolha != 'S' && escolha != 'N');
+
+
+	return escolha == 'S';
```

Comment:
Acho que seria interessante validar a entrada (não sei o quão criteriosos os professores vão ser quanto a isso). Nesse caso, qualquer coisa que não seja 'S' é considerada 'N'.

Reply:
A validação tá sendo feita na condição do do-while. Enquanto for diferente de 'S' ou de 'N', o jogo continua pedindo a entrada. Daí no return é só pra saber se a jogada vai ser automática ou não, mas quando chega lá, já vai ser garantido que 'escolha' é 'S' ou 'N'.

Reply:
Ahhh sim, show de bola.